operators/community-windows-machine-config-operator/3.0.0/manifests/community-windows-machine-config-operator.3.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,320 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/community-windows-machine-config-operator:community-4.8-5a0c01f
+    createdAt: 2021-08-05T18:20:00Z
+    description: An operator that enables Windows container workloads on OKD/OCP
+    operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
+    operators.operatorframework.io/builder: operator-sdk-v2.0.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: ""
+    olm.skipRange: ">=2.0.0 <3.0.0"
+  name: community-windows-machine-config-operator.v3.0.0
+  namespace: openshift-windows-machine-config-operator
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned: []
+  description: |-
+    ## WARNING
+    Community distribution of the Windows Machine Config Operator.
+
+    This is an preview build, and is not meant for production workloads.
+
+    Issues with this distribution of WMCO can be opened against the [WMCO repository](https://github.com/openshift/windows-machine-config-operator).
+    Please read through the [troubleshooting doc](https://github.com/openshift/windows-machine-config-operator/blob/community-4.8/docs/TROUBLESHOOTING.md)
+    before opening an issue
+
+    Please ensure that when installing this operator the startingCSV you subscribe to is supported on the
+    version of OKD/OCP you are using. This CSV is meant for OKD/OCP 4.8.
+
+    ## Documentation
+
+    ### Introduction
+    The Windows Machine Config Operator configures Windows instances into nodes, enabling Windows container workloads
+    to be ran within OKD/OCP clusters. Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/4.5/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
+    or by specifying existing instances through a ConfigMap. Through either method, the Windows instance must have the
```

Comment:
where are we expanding on the configMap method? should we add a link from the README here.

Reply:
I too suggest adding a link to https://github.com/openshift/windows-machine-config-operator#configuring-byoh-bring-your-own-host-windows-instances